x/auth/client/cli/tx_sign.go

```diff
@@ -89,7 +93,14 @@ func makeSignBatchCmd() func(cmd *cobra.Command, args []string) error {
 		}
 		scanner := authclient.NewBatchScanner(txCfg, infile)
 
-		for sequence := txFactory.Sequence(); scanner.Scan(); sequence++ {
+		if !clientCtx.Offline {
+			seqValue = 0
+			txFactory = txFactory.WithAccountNumber(0)
```

Comment:
If we set `txFactory = txFactory.WithSequence(0)`, can we avoid modifying the loop below?

Reply:
If we set it that way, then we'll have sequence number from `0` everytime, which overrides the sequence number provided, when in offline mode.